src/node_crypto.cc

```diff
@@ -5682,6 +5684,14 @@ bool DiffieHellman::Init(const char* p, int p_len, const char* g, int g_len) {
   return VerifyContext();
 }
 
+inline const modp_group* FindDiffieHellmanGroup(const char* name) {
+  for (size_t i = 0; i < arraysize(modp_groups); ++i) {
+    const modp_group* it = modp_groups + i;
```

Comment:
If you’re going to refer to each entry with a variable anyway you should also be able to just go with 

```c++
for (const modp_group& group : modp_groups) {
  ...
}
```

:slightly_smiling_face:

Reply:
Yep, just copied the logic from the existing code, will do! Thanks! :)